server/migrations/20181016125446-create-report.js

```diff
@@ -10,19 +10,19 @@ module.exports = {
     userId: {
       allowNull: false,
       type: Sequelize.INTEGER,
+      onDelete: 'CASCADE',
```

Comment:
This allows a reported article to be deleted and also delete its reference on the report table